getting_started/brainvision.md

```diff
@@ -50,24 +50,26 @@ For validation of BrainVision file triplets, you can use the [brainvision-valida
 
 ## Preprocessing of raw EEG data
 
-FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .eeg, .vmrk). For reading the data into FieldTrip you can refer to the .eeg file, for exampl
+FieldTrip needs the user to define what file to read in. The BrainVision Recorder software usually stores different filetypes (.vhdr, .dat, .vmrk). For reading the data into FieldTrip you can refer to the .dat file, for example. In order for FieldTrip to read the data file, it has to be exported as a .dat file from BrainVision.
 
     cfg = [];
-    cfg.dataset = '/users/karlheinz/EEG/myrecording.eeg';
+    cfg.dataset = '/users/karlheinz/EEG/myrecording.dat';
     ...
 
-The .eeg files are the raw data files, i.e. they contain the data as it has been stored upon acquisition.
```

Comment:
same as above, in my experience raw data are stored as '.eeg' not as '.dat'

Reply:
Ok, actually I was assuming that the whole section was involving data after export from BrainVision, in which case the choice between ".dat" and ".eeg" was already made before. But you are right, let's leave it for the later sections.